2023/assignments/aryan/aryan.md

```diff
@@ -0,0 +1,145 @@
+# <p align='center'>Git Workshop 2023 <p>
+
+### <p align='center'>Aryan Silawal</p>
+
+<p align="center">
+    <a href="#" target="blank"
+    ><img
+        align="center"
+        src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/facebook.svg"
+        alt="your_name"
+        height="24"
+        width="32"
+    /></a>
+    <a href="#" target="blank"
+    ><img
+        align="center"
+        src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/linked-in-alt.svg"
+        alt="your_name"
+        height="24"
+        width="32"
+    /></a>
+    <a href="#" target="blank"
+    ><img
+        align="center"
+        src="https://raw.githubusercontent.com/rahuldkjain/github-profile-readme-generator/master/src/images/icons/Social/github.svg"
+        alt="your_name"
+        height="24"
+        width="32"
+    /></a>
+       <a href="#" target="blank"
```

Comment:
`href='#'` replace **#** with your instagram profile link

Reply:
Please resolve this issue, 
If you donot have instagram id simple remove this `<a>` tag